lib/src/main/java/org/team100/lib/localization/NotePosition24ArrayListener.java

```diff
@@ -57,12 +64,12 @@ void consumeValues(NetworkTableEvent e) {
      * @return The x position in the camera in pixels, 0 should be the left of the
      *         screen
      */
-    public Optional<Double> getX() {
+    private Optional<Double> getX() {
```

Comment:
remove the "positions" field and just pass what you need to this function.  it is generally a bad idea for one method to stash something in a field so that another method can pick it up.  just pass it as an argument.

also what does this represent?  the comment says it's pixels.  but it's used below as some sort of yaw measurement.